client/src/i18n/en/form.json

```diff
@@ -121,7 +121,7 @@
             "RECORD_ERROR": "Please resolve any errors on the form (marked in red).",
             "REQUIRED": "This value is required.",
             "INVALID": "invalid data",
-            "LOCKED_USER": "You are locked, contact the administrator of the System",
+            "LOCKED_USER": "You are locked. Contact the system administrator",
```

Comment:
This message is shown when a user is locked out of the system because their account has been closed.  Let's say:
> Your account has been closed.  Please contact the system administrator to reopen your account.